.github/workflows/link-schedule.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build workflow
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  build-and-check-links:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@main
+
+      - name: Use Node.js 17.8.x
+        uses: actions/setup-node@main
+        with:
+          node-version: 17.8.x
+      - name: Build Docusaurus website
+        run: |
+          npm install
+          npm run build
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.5.0
```

Comment:
Oh, I don't think this would work...

Reply:
Got it, thank you. I'll use cmd line to refactor it and send a new review request after finished.